### ISSUE TITLE
pymedusa is redownloading repack of show when I already have another version that is preferred

### ISSUE THREAD

**Issue by marcmv1** [pymedusa/Medusa#3605]
Steps to reproduce:

Show Chicago PD
Configured with preferred word: amzn
Configured with undesired word: tbs

I had already downloaded this file:

Chicago.PD.S05E11.1080p.AMZN.WEB-DL.DDP5.1.H.264-KiNGS

Pymedusa saw that this was available:

Chicago.PD.S05E11.REPACK.1080p.WEB.x264-TBS[rartv]

Even though it was a different version of the file and undesired word TBS is included in the file name, it still overwrote the preferred version that I already had downloaded due to the word REPACK

IMO the word REPACK should not override undesired words when you already have a file downloaded with preferred words.

**Comment by Artwebb1986** [comment]
Yes repack should. Since 99% of the time a repack is fixing a screwed up release.

**Comment by marcmv1** [comment]
If it's the same release then I agree with you. But if I have an amzn and the repack is TBS I don't think my preferred version should be overwritten. Especially if it's an undesired released according to my config

**Comment by fernandog** [comment]
@marcmv1 we already implemented this check:  https://github.com/pymedusa/Medusa/pull/3235

Need full logs

**Comment by Artwebb1986** [comment]
You have KiNGS, webDL and WEB are in the same category unfortunately.  Could always set it to archived when you have your desired releases.

**Comment by fernandog** [comment]
@duramato do you have issues with this?

**Comment by marcmv1** [comment]
2018-01-12 16:23:26 INFO     FINDPROPERS :: [7b9111e] Beginning the search for new propers
2018-01-12 16:23:26 INFO     FINDPROPERS :: [7b9111e] Using proper search days: 2
2018-01-12 16:23:26 INFO     FINDPROPERS :: [OZNZB] :: [7b9111e] Searching for any new PROPER releases from OZNZB
2018-01-12 16:26:03 INFO     FINDPROPERS :: [NZB_PLANET] :: [7b9111e] Searching for any new PROPER releases from NZB_PLANET
2018-01-12 16:27:50 INFO     FINDPROPERS :: [Rarbg] :: [7b9111e] Searching for any new PROPER releases from Rarbg
2018-01-12 16:29:50 INFO     FINDPROPERS :: [7b9111e] Quality of Chicago.PD.S05E11.REPACK.1080p.WEB.x264-TBS[rartv] is 1080p WEB-DL
2018-01-12 16:29:50 INFO     FINDPROPERS :: [7b9111e] Preferring Chicago.PD.S05E11.REPACK.1080p.WEB.x264-TBS[rartv] (repack/proper/real/rerip over nuked)
2018-01-12 16:29:50 INFO     FINDPROPERS :: [7b9111e] Found a desired proper: Chicago.PD.S05E11.REPACK.1080p.WEB.x264-TBS[rartv]
2018-01-12 16:29:50 INFO     FINDPROPERS :: [7b9111e] Quality of chicago.pd.s05e11.repack.1080p.web.x264-tbs-Pre is 1080p WEB-DL
2018-01-12 16:29:50 INFO     FINDPROPERS :: [7b9111e] Preferring chicago.pd.s05e11.repack.1080p.web.x264-tbs-Pre (repack/proper/real/rerip over nuked)
2018-01-12 16:29:53 INFO     FINDPROPERS :: [7b9111e] Completed the search for new propers, next check in approx. 38m, 32s
2018-01-12 16:29:58 INFO     SEARCHQUEUE-DAILY-SEARCH :: [7b9111e] Beginning daily search for new episodes
2018-01-12 16:29:59 INFO     SEARCHQUEUE-DAILY-SEARCH :: [7b9111e] Using daily search providers
2018-01-12 16:30:10 INFO     SEARCHQUEUE-DAILY-SEARCH :: [OZNZB] :: [7b9111e] Removing cache entries older than 30 days from oznzb
2018-01-12 16:30:17 INFO     SEARCHQUEUE-DAILY-SEARCH :: [NZB_PLANET] :: [7b9111e] Removing cache entries older than 30 days from nzb_planet
2018-01-12 16:30:25 INFO     SEARCHQUEUE-DAILY-SEARCH :: [Rarbg] :: [7b9111e] Removing cache entries older than 30 days from rarbg
2018-01-12 16:30:28 INFO     SEARCHQUEUE-DAILY-SEARCH :: [7b9111e] No needed episodes found
2018-01-12 16:31:55 INFO     Thread-30 :: [7b9111e] Broken providers found: [u'torrentproject']
2018-01-12 16:31:55 INFO     TORNADO :: [7b9111e] User logged into the Medusa web interface
2018-01-12 16:31:56 INFO     Thread-26 :: [7b9111e] Processing C:\Marc\Torrents\Completed\tv\Chicago.PD.S05E11.REPACK.1080p.WEB.x264-TBS[rarbg]\chicago.pd.s05e11.repack.1080p.web.x264-tbs.mkv
2018-01-12 16:31:56 INFO     Thread-26 :: [7b9111e] Old file: C:\Marc\TV\Chicago P.D\Chicago.P.D.S05E11.Confidential.1080p.WEB-DL.h264-KiNGS-Pre.mkv
2018-01-12 16:31:56 INFO     Thread-26 :: [7b9111e] New file: C:\Marc\Torrents\Completed\tv\Chicago.PD.S05E11.REPACK.1080p.WEB.x264-TBS[rarbg]\chicago.pd.s05e11.repack.1080p.web.x264-tbs.mkv
2018-01-12 16:31:56 INFO     Thread-26 :: [7b9111e] Old size: 2.90 GB
2018-01-12 16:31:56 INFO     Thread-26 :: [7b9111e] New size: 1.25 GB
2018-01-12 16:31:56 INFO     Thread-26 :: [7b9111e] New file is 1.65 GB smaller
2018-01-12 16:31:56 INFO     Thread-26 :: [7b9111e] This download is marked a priority download so I'm going to replace an existing file if I find one
2018-01-12 16:31:56 INFO     Thread-26 :: [7b9111e] Trying to clean any empty folder under C:\Marc\TV\Chicago P.D
2018-01-12 16:31:56 INFO     Thread-26 :: [7b9111e] Using nzb_name for release name.
2018-01-12 16:31:57 INFO     Thread-26 :: [7b9111e] Processing succeeded for C:\Marc\Torrents\Completed\tv\Chicago.PD.S05E11.REPACK.1080p.WEB.x264-TBS[rarbg]\chicago.pd.s05e11.repack.1080p.web.x264-tbs.mkv
2018-01-12 16:31:57 INFO     Thread-26 :: [7b9111e] Successfully processed.

**Comment by duramato** [comment]
Haven't experienced this @fernandog

**Comment by fernandog** [comment]
@marcmv1 you need to enable DEBUG logs in settings and reproduce the issue again
